metrics/Software_Bill_of_Materials.md

```diff
@@ -44,21 +43,17 @@ ExtractedText: nonprofit
 LicenseCrossReference:
 LicenseComment: found by nomos
 ```
-A full text file for this scan may be found here:
-https://drive.google.com/file/d/18ClTeQo64sU8dVzSQDMYzO6jLHHa9TPg/view?usp=sharing
 
+### Tools Providing the 
+[DoSOCSv2](https://github.com/DoSOCSv2/DoSOCSv2) embedded as an [Augur](https://github.com/chaoss/augur) Service.
 
-## Reference Implementation
 A file by file SBOM is available with Augur configured using the DoSOCSv2 plugin.  The relevant parts of the database schema are illustrated below. The most important points, from an SBOM perspective, is simpler than the software licensing metrics described elsewhere.  For the SBOM, we simply look at the enumeration of:
 * Packages
 * Package_Files
 * Files (which may be, but are unlikely to be also included in other packages).
 License information is included as part of an SBOM, but the complexity of license identification is clarified in the [License_Count](https://github.com/chaoss/wg-risk/blob/master/metrics/License_Count.md), [License_Coverage](https://github.com/chaoss/wg-risk/blob/master/metrics/License_Coverage.md), and [License_Declared](https://github.com/chaoss/wg-risk/blob/master/metrics/License_Declared.md) metrics.
 ![](https://github.com/chaoss/wg-risk/blob/master/metrics/images/SBOM.png)
 
-## Known Implementations
-* [DoSOCSv2](https://github.com/DoSOCSv2/DoSOCSv2) embedded as an [Augur](https://github.com/chaoss/augur) Service.
-
 ## Resources

```

Comment:
line 57: The [new Metric Template](https://github.com/chaoss/metrics/blob/master/activity-metrics/metric-template.md) replaced "Resources" with "References".